src/components/DropZone/tests/DropZone.test.tsx

```diff
@@ -330,11 +331,11 @@ function createEvent(name: string, files: any) {
   return evt;
 }
 
-function setBoundingClientRect(size: keyof typeof widths) {
+function setBoundingClientRect(size: Width) {
   Element.prototype.getBoundingClientRect = jest.fn(() => {
     return {
-      width: widths[size],
-      height: 100,
+      width: size,
+      height: size,
```

Comment:
It seems weird to use a Width enum as the type of size, which is then used for height.